test/Owned.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Owned = artifacts.require('Owned');
+const truffleAssert = require('truffle-assertions');
+
+contract('Owned', (accounts) => {
+    let instance;
+
+    // Initiating new contract before each test
+    beforeEach("Creating new contract", async () => {
+        instance = await Owned.new({from: accounts[0]});
+    });
+
+    // Testing getOwner function.
+    it('Owner should be ' + accounts[0], () => {
+        return instance.getOwner()
+            .then(owner => {
+                assert.strictEqual(owner, accounts[0]);
+            });
+    });
+
+    // Testing changeOwner function.
+    it('Owner should be ' + accounts[1], () => {
+        return instance.changeOwner(accounts[1], {from: accounts[0]})
+            .then(() => {
```

Comment:
Testing Owned is not really the point of this repo here, but you could check the receipt too.